cli/command/load_balancer.go

```diff
@@ -10,6 +18,497 @@ func NewLoadBalancerCommand(b *CommandBase) *LoadBalancerCommand {
+}
+
+func (l *LoadBalancerCommand) healthcheck(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "LOADBALANCER_NAME")
+	if err != nil {
+		return err
+	}
+
+	requiresUpdate := false
+	healthCheck := models.HealthCheck{}
+
+	if target := c.String("healthcheck-target"); target != "" {
+		target, err := parseTarget(target)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.Target = target
+	}
+
+	if interval := c.String("healthcheck-interval"); interval != "" {
+		interval, err := strconv.Atoi(interval)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.Interval = interval
+	}
+
+	if timeout := c.String("healthcheck-timeout"); timeout != "" {
+		timeout, err := strconv.Atoi(timeout)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.Timeout = timeout
+	}
+
+	if healthyThreshold := c.String("healthy-threshold"); healthyThreshold != "" {
+		healthyThreshold, err := strconv.Atoi(healthyThreshold)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.HealthyThreshold = healthyThreshold
+	}
+
+	if unhealthyThreshold := c.String("unhealthy-threshold"); unhealthyThreshold != "" {
+		unhealthyThreshold, err := strconv.Atoi(unhealthyThreshold)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.UnhealthyThreshold = unhealthyThreshold
+	}
+
+	loadBalancerID, err := l.resolveSingleEntityIDHelper("load_balancer", args["LOADBALANCER_NAME"])
+	if err != nil {
+		return err
+	}
+
+	loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+	if err != nil {
+		return err
+	}
+
+	if requiresUpdate {
+		if healthCheck.Target != "" {
+			loadBalancer.HealthCheck.Target = healthCheck.Target
+		}
+
+		if healthCheck.Interval != 0 {
+			loadBalancer.HealthCheck.Interval = healthCheck.Interval
+		}
+
+		if healthCheck.Timeout != 0 {
+			loadBalancer.HealthCheck.Timeout = healthCheck.Timeout
+		}
+
+		if healthCheck.HealthyThreshold != 0 {
+			loadBalancer.HealthCheck.HealthyThreshold = healthCheck.HealthyThreshold
+		}
+
+		if healthCheck.UnhealthyThreshold != 0 {
+			loadBalancer.HealthCheck.UnhealthyThreshold = healthCheck.UnhealthyThreshold
+		}
+
+		req := models.UpdateLoadBalancerRequest{
+			LoadBalancerID: loadBalancerID,
+			Ports:          &loadBalancer.Ports,
+			HealthCheck:    &loadBalancer.HealthCheck,
+		}
+
+		jobID, err := l.client.UpdateLoadBalancer(req)
+		if err != nil {
+			return err
+		}
+
+		onCompleteFn := func(loadBalancerID string) error {
+			loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+			if err != nil {
+				return err
+			}
+
+			l.printer.StopSpinner()
+			return l.printer.PrintLoadBalancerHealthCheck(loadBalancer)
+		}
+
+		return l.waitOnJobHelper(c, jobID, "Updating health check on load balancer", onCompleteFn)
+
+	} else {
+		l.printer.Printf("No health check update required. Displaying load balancer health check.\n")
```

Comment:
Don't print this here since that would mess up someones script. 
This function is kind of odd in that it is both a read and an update - maybe we should rethink that ux?

Reply:
This was fleshed out explicitly mainly because of what the help message was prescribing:

https://github.com/quintilesims/layer0/blob/develop/cli/command/load_balancer_command.go#L108
`view or update the health check for a load balancer`

I think there could be room for discussion about how to make both of these options available for the user.